charts/sonarqube/templates/deployment.yaml

```diff
@@ -174,6 +174,10 @@ spec:
         - name: "wait-for-db"
           image: {{ default "busybox:1.32" .Values.initContainers.image }}
           imagePullPolicy: {{ .Values.image.pullPolicy  }}
+          {{- if $securityContext := .Values.initContainers.securityContext }}
```

Comment:
reusing the `initContainers` scope is misleading

Reply:
Hi! Isn't this a initContainer? Should I create a value like wait-for-db.securityContext ?